cypress/locators/portrait/index.js

```diff
@@ -1,4 +1,4 @@
-import { PORTRAIT_PREVIEW, PORTRAIT_INITIALS } from "./locators";
+import { PORTRAIT_PREVIEW, PORTRAIT_INITIALS } from "../../../playwright/components/portrait/locators";
```

Comment:
**comment** - I'm not sure what's going on here. Why do we have **cypress** locators that are importing stuff from the corresponding **playwright** locators?

As far as I can tell from a quick search, we don't seem to be importing the cypress portrait locators into any Cypress tests any more (but I might have missed something!) - if so then we can just delete this file and the `cypress/locators/portrait` folder. But if not then we should keep the `cypress/locators/portrait/locators.ts` file as well (just those locators that we need) and this file should continue to import what it needs from there.

Reply:
Good point, i'll just delete the file